meta-hailo/kas/opt/hailo.inc.lock.yml

```diff
@@ -0,0 +1,6 @@
+header:
+    version: 14
+overrides:
+    repos:
+        isar:
+            commit: 9dc362cd7115074c0f60843e687ed89db5c9339a
```

Comment:
What's this lock file for?

Reply:
This file originates from Flex. I left it here to maintain consistency. However, I can remove all the unused files if preferred.

Reply:
You don't use the kas lock feature in this layer, so simply drop this. It will only cause problems when trying to update isar via the main kas file.

Reply:
ACK